src/mw-ui-components/directives/mw_tab_bar.js

```diff
@@ -1,61 +1,118 @@
 angular.module('mwUI.UiComponents')
-  //TODO rename
-  /**
-   * @example ```html
-   * <!-- change callback example -->
-   * <div mw-tabs active-pane-number="myCtrl.activePane" tab-changed="myCtrl.tabChanged">
-      <div mw-tabs-pane="{{'mytitle'| i18n}}">
-      Tab 1
-      </div>
-      <div mw-tabs-pane="{{'mytitle_2'| i18n}}">
-      Tab 2
-      </div>
-    </div>
-   * ```
-   */
-  .directive('mwTabs', function () {
+//TODO rename
+/**
+ * @example ```html
+ * <!-- change callback example -->
+ * <div mw-tabs active-pane-number="myCtrl.activePane" tab-changed="myCtrl.tabChanged">
+ <div mw-tabs-pane="{{'mytitle'| i18n}}">
+ Tab 1
+ </div>
+ <div mw-tabs-pane="{{'mytitle_2'| i18n}}">
+ Tab 2
+ </div>
+ </div>
+ * ```
+ */
+  .directive('mwTabs', function ($rootScope) {
     return {
       transclude: true,
       scope: {
-        justified: '=',
-        activePaneNumber: '=',
-        tabChanged: '='
+        justified: '=?',
```

Comment:
i'm missing the definition for the scope attribute panes

Reply:
Sorry i don't know what you mean. Which attributes are you missing?
The attributes for the pane (like title, icon,...) are defined on the pane itself

Reply:
the `scope.panes` array is only used internally and you can not set it from the outside. It is the array where each registration of `mw-tab-bar` is stored.

```html
<div mw-tabs>
  <div mw-tabs-pane="Pane 1"></div>
  <div mw-tabs-pane="Pane 2"></div>
</div>
```